pkg/Microsoft.NETCore.Targets/Microsoft.NETCore.Targets.pkgproj

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
+  <PropertyGroup>
+    <Version>1.0.1</Version>
+    <IsLineupPackage>true</IsLineupPackage>
+    <RuntimeFileSource>runtime.json</RuntimeFileSource>
+  </PropertyGroup>
+  
+  <Import Project="..\NETStandard.Library\NETStandard.Library.packages.targets" />
+  
+  <ItemGroup>
+    <Package Include="System.Private.Uri" />
+    <NativePackage Include="runtime.native.System.IO.Compression" />
+    <NativePackage Include="runtime.native.System" />
+    <NativePackage Include="runtime.native.System.Net.Http" />
+    <NativePackage Include="runtime.native.System.Security.Cryptography" />
+    <NativePackage Include="runtime.native.System.Net.Security" />
+  </ItemGroup>
```

Comment:
Right now we don't put the runtime package in the lineup.  Instead it carries its own runtime.json.  If we are going to put the runtime package in the lineup we need to come up with a way to plumb the runtime.json info across repos.  We do not want to have two packages with the same runtime.json entries since that creates versioning issues.